screens/Landmark.js

```diff
@@ -37,6 +43,33 @@ class LandmarkScreen extends React.Component {
     }
   }
 
+  componentDidUpdate(prevProps, prevState){
+    if (this.props.landmarkDetails.name !== prevProps.landmarkDetails.name){
+      this.setState({seeMoreSelected: false})
+    }
+  }
+
+  getDirectionsToLandmark = landmarkCoordinates => {
+    this.props.fetchDirections(landmarkCoordinates);
+    this.props.closeDrawer();
+  };
+
+  getDescription = (type) => {
+    const { landmarkDetails } = this.props
+    let description = landmarkDetails.description;
+    let descriptionMaxWordLength = 500;
+    let shortDescriptionMaxWordLength = 100;
+
+    if (type === 'teaser') {
```

Comment:
Heads up that eventually the short description will be a different chunk of text than the description, so we will need to refactor to use that once we have all the fields/data in the db. But this works great for now so we can test visually.

Reply:
Yup, all good. Should be an easy thing to add.